modules/manta/germline/main.nf

```diff
@@ -26,13 +26,14 @@ process MANTA_GERMLINE {
     script:
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
-    def options_manta = target_bed ? "--exome --callRegions $target_bed" : ""
+    def options_manta = target_bed ? "--callRegions $target_bed" : ""
     """
     configManta.py \
-        --bam $input \
+        --bam ${input.join(' --bam ')} \
```

Comment:
don't we get 2 `--bam` in a row if we keep the first `--bam` here?

Reply:
We get additional `--bam` parameters in the command only when we have two or more bam files in the list. You can see that first of three tests for this module only uses a single cram file and it works :)